### DISCUSSION TITLE
Welcome to VsCode-Python-Extensions Discussions!

### DISCUSSION THREAD

**Discussion by echoWebNerds** [echoWebNerds/VsCode-Python-Extensions#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by granjacours410-source** [comment]
Echo nerd Python repo.hole my studio is Almost disabeld for debugging
